src/ExadelMentorship.IntegrationTests/DIConfigurer.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ExadelMentorship.IntegrationTests
+{
+    public class DIConfigurer
+    {
+        private ServiceProvider serviceProvider;
+        public DIConfigurer()
+        {
+            serviceProvider = new ServiceCollection()
```

Comment:
Constructors should be simple i.e. assign dependencies to fields. Move it to `.Congifure(): IServiceProvider` method